tree/treeplayer/src/TTreeProcessorMT.cxx

```diff
@@ -491,6 +491,7 @@ void TTreeProcessorMT::Process(std::function<void(TTreeReader &)> func)
    const std::vector<Internal::NameAlias> &friendNames = fFriendInfo.fFriendNames;
    const std::vector<std::vector<std::string>> &friendFileNames = fFriendInfo.fFriendFileNames;
 
+   TThreadExecutor pool;
```

Comment:
I would rather EnableImplicitMT until TTreeProcessorMT is adapted for explicit multithreading. 

This may work, but you don't have any reason to maintain an instance of TThreadExecutor when what you want is to set up a pool of threads.

Reply:
Could you elaborate? Where does the pool come from? Is it created by `EnableImplicitMT` upfront if you call it? And why is creating an instance of `TThreadExecutor` bad in `TTreeProcessorMT`?

Reply:
Please someone (@xvallspl @bluehood @stwunsch) tell me exactly what to do: everybody in this discussion seems to have a different opinion. From my point of view, the logic is wrong in this code anyway. But I don't have any strong opinion about how to fix the issue, and to be honest I don't really care. I'm in this loop only because it make a test crashing on Windows... I can also close this PR and let you guys decide how to fix this issue...

Reply:
@bellenot please bear with us a little while we figure it out :smile:

Reply:
As, I commented during lunch, I mistook the file I was commenting on :man_facepalming: .

Reply:
As I commented during lunch, I mistook the file I was commenting on :man_facepalming: